src/main.rs

```diff
@@ -533,77 +633,20 @@ fn main() -> ! {
     spi_drv.init();
     spi_drv.reset(&mut delay);
 
-    // Turn the ESP32's onboard multi-color LED off
-    set_led(&mut spi_drv, &mut uart, 0, 0, 0);
-    delay.delay_ms(500);
```

Comment:
Keep these lines, they're useful to make sure the ESP32 LED is turned off to begin with.

Reply:
Resolved in [following commit](https://github.com/Jim-Hodapp-Coaching/esp32-pico-wifi/pull/4/commits/c10bb540c941bdb44dbff93ce28af3736f1ca7b1)